org.osgi.test.assertj.cm/src/main/java/org/osgi/test/assertj/cm/configurationevent/ConfigurationEventConditions.java

```diff
@@ -0,0 +1,160 @@
+/*******************************************************************************
+ * Copyright (c) Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *******************************************************************************/
+
+package org.osgi.test.assertj.cm.configurationevent;
+
+import static org.assertj.core.api.Assertions.allOf;
+import static org.assertj.core.api.Assertions.anyOf;
+import static org.assertj.core.api.Assertions.not;
+
+import java.util.Objects;
+
+import org.assertj.core.api.Condition;
+import org.assertj.core.condition.VerboseCondition;
+import org.osgi.service.cm.Configuration;
+import org.osgi.service.cm.ConfigurationEvent;
+
+/**
+ * A Utility-Class thats Provides public static methods to create
+ * {@link Condition}s for an <code>ConfigurationEvent</code>}
+ */
+public final class ConfigurationEventConditions {
```

Comment:
Should this be default visibility, or is it user facing? Also applies to all the methods